sapi/cli/php_cli.c

```diff
@@ -643,6 +643,12 @@ static int do_cli(int argc, char **argv) /* {{{ */
 #endif
 #ifdef HAVE_GCOV
 					" GCOV"
+#endif
+					,
+#ifdef PHP_BUILD_PROVIDER
+					"This build by " PHP_BUILD_PROVIDER "\n"
```

Comment:
```suggestion
					"Built by " PHP_BUILD_PROVIDER "\n"
```

Perhaps this sounds more grammatically correct.